x/fungible/types/keys.go

```diff
@@ -27,17 +27,14 @@ func KeyPrefix(p string) []byte {
 }
 
 var (
-	ModuleAddress = authtypes.NewModuleAddress(ModuleName)
-	//ModuleAddressEVM common.EVMAddress
+	ModuleAddress    = authtypes.NewModuleAddress(ModuleName)
 	ModuleAddressEVM = common.BytesToAddress(ModuleAddress.Bytes())
-	AdminAddress     = "zeta1rx9r8hff0adaqhr5tuadkzj4e7ns2ntg446vtt"
+	// ModuleAddressZEVM is calculated in the same way as ModuleAddressEVM.
+	// Maintain it for legibility in functions calling fungible module address from zEVM.
+	ModuleAddressZEVM = common.BytesToAddress(ModuleAddress.Bytes())
+	AdminAddress      = "zeta1rx9r8hff0adaqhr5tuadkzj4e7ns2ntg446vtt"
```

Comment:
What is this used for ?

Reply:
That `AdminAddress` was actually there before this PR. It appears as a change as I removed some unused code like the init() function.

Reply:
Removed in f20129df6881a8c92d6fc46e4ef56479823a2177, confirmed it's unused.